# config file to specify the file location of the robots urdf calibration values
# and the name used to the generic calibration variable name

#welder
kr8_r1420_rcb/welder_tool_surface_kuka_cal: 
  file_location: "kuka/kuka_manipulators/kuka_common_support/urdf/kr8_calibration_values.urdf.xacro"
  calibration_value_names:
    welder_tip_x: 'calibration_x'
    welder_tip_y: 'calibration_y'
    welder_tip_z: 'calibration_z'
    welder_tip_roll: 'calibration_roll'
    welder_tip_pitch: 'calibration_pitch'
    welder_tip_yaw: 'calibration_yaw'

#2d laser
kr8_r1420_rcb/2d_laser_surface_kuka_cal:
  file_location: "kuka/kuka_manipulators/kuka_common_support/urdf/kr8_calibration_values.urdf.xacro"
  calibration_value_names:
    laser_2D_x: 'calibration_x'
    laser_2D_y: 'calibration_y'
    laser_2D_z: 'calibration_z'
    laser_2D_roll: 'calibration_roll'
    laser_2D_pitch: 'calibration_pitch'
    laser_2D_yaw: 'calibration_yaw'

# TODO: copy above layout for other tool surfaces
